src/index.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+const fetch = require('node-fetch');
+const chalk = require('chalk');
+const argv = require('./yargs').argv;
+
+let validated = [];
+function validateLinks(links) {
+    console.log(chalk.cyanBright.bold('hola mundo si estas en la funcion ValidateLinks')); // con este se que ya esta accediendo a la funcion OPTIONMENU
```

Comment:
documentación en ingles
```suggestion
    console.log(chalk.cyanBright.bold('hola mundo si estas en la funcion ValidateLinks')); 
```